### ISSUE TITLE
Messages not send in android sdk 

### ISSUE THREAD

**Issue by abdulmoeed137** [chat21/chat21-android-sdk#27]
Hello,
I have cloned the the project and added my json file generated from firebase with correct package name but my messages are not sending

**Comment by andrealeo83** [comment]
Have you installed https://github.com/chat21/chat21-cloud-functions as described on README?

**Comment by andrealeo83** [comment]
Please describe your code

**Comment by abdulmoeed137** [comment]
No. I think it's for web? Is it necessary to use this in android also?

**Comment by andrealeo83** [comment]
Yes please read this : https://github.com/chat21/chat21-android-sdk#pre-requisites

**Comment by abdulmoeed137** [comment]
Hi,
Thank you for your help.
Can you please help me what should I add in android sdk to solve this
issue.

**Comment by abdulmoeed137** [comment]
I think it's for web. I am not able to find its android version.

**Comment by ajmalkhan1998** [comment]
you need to upload those cloud settings to your firebase(firebase cloud functioning),then only the messages will be sent.
